src/main/java/seedu/duke/commands/ClearCommand.java

```diff
@@ -0,0 +1,18 @@
+package seedu.duke.commands;
+
+import seedu.duke.data.Budget;
+import seedu.duke.data.ShoppingList;
+
+public class ClearCommand extends Command {
+
+    public static final String COMMAND_WORD = "CLEAR";
+
+    public static final String LIST_CLEARED_MESSAGE = "Your shopping list has been cleared.\n";
+
+    @Override
+    public CommandResult execute(ShoppingList items, Budget myBudget) {
+        items.clearList();
+        String feedback = LIST_CLEARED_MESSAGE;
```

Comment:
Maybe we can change all our feedback messages, whether failed or successful to be of the same format? Eg. SUCCESS_MESSAGE/MESSAGE_SUCCESS? :D))

Reply:
Ok, noted!!